README.md

```diff
@@ -410,26 +410,6 @@ AWSDDLog.add(AWSDDTTYLogger.sharedInstance()) // TTY = Xcode console
 [AWSDDLog addLogger:[AWSDDTTYLogger sharedInstance]]; // TTY = Xcode console
 ```
 
-## Install the Reference Documentation in Xcode
```

Comment:
Why are we removing this?

Reply:
Xcode hasn't supported installing docsets for custom frameworks/classes for several versions now (since 8 I think?) which was underscored by removing docsetutil in 9.